network/framework/src/noise/handshake.rs

```diff
@@ -74,64 +74,6 @@ impl AntiReplayTimestamps {
     }
 }
 
-/// Noise handshake authentication mode.
-pub enum HandshakeAuthMode {
```

Comment:
If I understand this change correctly, you could probably separate this into a different commit?

Reply:
Yeah, that's one of the earlier things I did and possibly separable